kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -458,11 +458,13 @@ public class BigQuerySinkConfig extends AbstractConfig {
           + "older than the specified partition expiration time will be permanently deleted. "
           + "Existing tables will not be altered to use this partition expiration time.";
 
-  public static final String GCP_CONNECTOR_PROVIDER_CONFIG = "provider";
-  private static final ConfigDef.Type GCP_CONNECTOR_PROVIDER_TYPE = ConfigDef.Type.STRING;
-  public static final String GCP_CONNECTOR_PROVIDER_DEFAULT = "Confluent Platform";
-  private static final ConfigDef.Importance GCP_CONNECTOR_PROVIDER_IMPORTANCE = ConfigDef.Importance.LOW;
-  public static final List<String> GCP_CONNECTOR_PROVIDER_TYPES = Stream.of("Confluent Platform", "Confluent Cloud")
+  //This config determines where the connector is hosted (Confluent Cloud or Confluent Platform).
+  //This is not enforced and defaulted to "Confluent Platform". Currently, it is only used for user-agent tracking in GCP.
+  public static final String CONNECTOR_RUNTIME_PROVIDER_CONFIG = "runtimeProvider";
```

Comment:
@mukkachaitanya do you think its a good idea to move this config and validator to connect-utils or some other common place so other connectors can just import the library and reuse it as is.

I feel like this is going to be needed in many connectors in team1 also (I already have a ticket for this).   I recently saw yash doing some common code in connect-utils for Retry library so that seems right place but I am not 100% sure so asking you.

Reply:
Unfortunately, I don't think this might fly for Open Source connectors as they don't use any of Confluent's proprietary libraries. We can surely try doing something for other proprietary connectors if we find something common. Another idea is to make this part of Connect Framework, but it might need a KIP.

Reply:
got it, we can copy-paste for now.  Making it part of connect framework seems too much.